rotkehlchen/chain/evm/decoding/oneinch/decoder.py

```diff
@@ -99,10 +107,3 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
         return {
             self.router_address: (self.decode_action,),
         }
-
-    def counterparties(self) -> list[CounterpartyDetails]:
-        return [CounterpartyDetails(
-            identifier=self.counterparty,
```

Comment:
This is the only thing that made it difficult for me as this was unexpected. You recently made this using `self` for somethign I expected to be static. Though I can understand why. Made some changes to make it work.